.claude/commands/BMad/tasks/validate-next-story.md

```
@@ -0,0 +1,140 @@
+# /validate-next-story Task
+
+When this command is used, execute the following task:
+
+<!-- Powered by BMAD™ Core -->
+
+# Validate Next Story Task
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)